Makefile

```diff
@@ -26,7 +26,7 @@ build:
 # Clients
 ################################################################################
 
-.PHONY: internal/client
+.PHONY: internal/client/client.gen.go
```

Comment:
unrelated: without this, you have to manually delete the client before re-generating it